x-pack/plugins/session_view/common/mock/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './app_context_render';
```

Comment:
Personally I'm not a fan of re-exporting like this. I know kibana does it a lot. It makes it hard to tell what imports came from which file when we start re-exporting from multiple files

I think we should just import exactly what we need.